src/sentry/models/scheduledjob.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import absolute_import, print_function
 
-from django.db import models
+from django.db import models, transaction
 from django.utils import timezone
 from jsonfield import JSONField
 
-from sentry.db.models import Model, sane_repr
+from sentry.db.models import (BaseManager, Model, sane_repr)
+
+
+class ScheduledJobManager(BaseManager):
+    def schedule_job(self, payload, job):
```

Comment:
Thanks for the comments- makes sense. Jobs here needs to be a tuple to accommodate both the name, payload, and the date scheduled for each job. For the first use I will always have two jobs i'm scheduling at a time which is why I went with the bulk_create instead of the single. Does that make sense?